package.json

```diff
@@ -24,7 +24,7 @@
     "tsc": "tsc --version && tsc -p .",
     "apply-next-version": "node utils/apply_next_version.js",
     "bundle": "npx browserify -r ./index.js:puppeteer -o utils/browser/puppeteer-web.js",
-    "test-types": "node utils/doclint/generate_types && npx -p typescript@3.2 tsc -p utils/doclint/generate_types/test/",
+    "test-types": "node utils/doclint/generate_types && npx -p typescript@3.8 tsc -p utils/doclint/generate_types/test/",
```

Comment:
Good catch. Shall we just drop the `@3.8` here, so that it uses the version down below in `package.json`?

Reply:
💯